src/app/Components/LanguageSources.svelte

```diff
@@ -0,0 +1,123 @@
+<script lang="ts">
+  import Button from "./Button.svelte";
+  export let sources: any[];
```

Comment:
Honestly, this interface needs to be in some file, probably in `src/common/...`, but this will do for now:

```suggestion
  interface DictionarySource {
      name: string;
      size: number;
      type: string;
  }

  export let sources: DictionarySource[];
```